src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = "Hello! I'm Duke\n" + "What can I do for you?";
+        String goodbye = "Bye. Hope to see you again soon!";
+        System.out.println(greeting);
+
+        Task[] tasklist = new Task[100];
+
+        Scanner scanner = new Scanner(System.in);
+        int flag = 0;
```

Comment:
variable names can be more descriptive to facilitate understanding. 
use boolean type instead of int type, phrase like a question eg. isExit